Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -252,6 +258,55 @@ def ip_to_indicator_type(ip):
             return None
 
 
+def auto_detect_indicator_type(indicator_value):
+    """Infer the type of the indicator.
+    Args:
+        indicator_value(str): The indicator whose type we want to check.
+    Returns:
+        str. The type of the indicator.
+    """
```

Comment:
please use this format:
"""Returns the indicator type of the input IP.
        :type ip: ``str``
        :param ip: IP address to get it's indicator type.
        
        :rtype: ``str``
        :return:: Indicator type from FeedIndicatorType, or None if invalid IP address.